File: app/src/main/java/com/compuspulse/data/remote/model/ProjectCategory.java
package com.compuspulse.data.remote.model;

public class ProjectCategory {
    private int id;
    private String name;

    public ProjectCategory() {
    }

    public ProjectCategory(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name == null ? "" : name;
    }
}

File: app/src/main/java/com/compuspulse/data/remote/model/ProjectCategory.kt
package com.compuspulse.data.remote.model

class ProjectCategory() {
    private var id: Int = 0
    private var name: String? = null

    constructor(id: Int, name: String?) : this() {
        this.id = id
        this.name = name
    }

    fun getId(): Int = id

    fun getName(): String = name ?: ""
}
